plopfile.js

```diff
@@ -86,23 +87,43 @@ module.exports = function (plop) {
         path: 'README.md',
         templateFile: 'templates/ROOT_README.hbs.md',
         force: true,
-        data: {jsPackageNames, gemNames},
+        data: {jsPackages, gems},
       },
     ],
   });
 };
 
-function getPackageNames(type = 'js') {
+function getPackages(type = 'js') {
   const packagesPath = path.join(
     __dirname,
     type === 'js' ? 'packages' : 'gems',
   );
-  return readdirSync(packagesPath).filter(packageName => {
+
+  return readdirSync(packagesPath).reduce((acc, packageName) => {
     const packageJSONPath = path.join(
       packagesPath,
       packageName,
       'package.json',
     );
-    return existsSync(packageJSONPath);
-  });
+
+    if (existsSync(packageJSONPath)) {
+      const {name, description} = require(packageJSONPath);
+
+      acc.push({name: name.replace('@shopify/', ''), description});
+    }
+
+    return acc;
+  }, []);
+}
+
+function validatePackageName(name) {
+  return (
+    /^(?:@[a-z0-9-*~][a-z0-9-*._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$/g.test(
```

Comment:
This regex was copied directly from the lint rule:

<img width="895" alt="Screen Shot 2020-06-12 at 11 55 01 AM" src="https://user-images.githubusercontent.com/46533681/84521847-9e145200-aca3-11ea-8225-6bb905677684.png">